package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ununifi-client",
-  "version": "0.46.1",
+  "version": "0.46.1-rc1",
```

Comment:
Since the latest version is "0.46.1", isn't this "0.46.2-rc1" even if you want to add rc tag?

Reply:
Gotit